pandas/core/algorithms.py

```diff
@@ -542,7 +543,7 @@ def value_counts(values, sort=True, ascending=False, normalize=False,
 
     else:
 
-        if is_categorical_dtype(values) or is_sparse(values):
+        if is_extension_array_dtype(values) or is_sparse(values):
```

Comment:
is_sparse should be recognized as an extension array type

Reply:
It's on my todo list. I'll do it after interval.